packages/@guardian/eslint-plugin-source-foundations/package.json

```diff
@@ -28,7 +29,7 @@
 	"devDependencies": {
 		"@guardian/eslint-config": "^1.0.0",
 		"@guardian/eslint-config-typescript": "^1.0.0",
-		"@guardian/libs": "^3.0.0",
+		"@guardian/libs": "^8.0.5",
```

Comment:
why this bump?

Reply:
Because otherwise the e2e test throws an error about a missing peer dep of `web-vitals` (see description). Also thought it would make it easier to use the `*workspace` option once the package is migrated as the latest is used in csnx. The other option is to add `web-vitals` as a dependency and then revert the version back to v3. What do you think @sndrs ?

Reply:
i'm not quite sure why this happens? for one thing, v8 of libs still has a peerdep on `web-vitals` https://github.com/guardian/csnx/blob/main/libs/%40guardian/libs/package.json#L17.

how does the missing peerdep affect the test (given it doens't consume the peerdep)?

Reply:
We chatted about this and established that the latest version of libs imports web vitals dynamically, which is why this error is resolved when we upgrade. We are going to bump libs in all Source packages in a separate PR.